manuscript/1-prehistory.md

```diff
@@ -14,4 +14,4 @@ Things initially went pretty well, I thought. They re-themed the site a bit, usi
 
 This unfortunately contributed to the deep level of community fragmentation that I'd hoped to help avoid. Another company, focused on their own corporate needs, bought the PowerShell.com domain name and launched a site. They initially dumped a lot of money into keeping it alive and steady, including paying people to be responsive in the forums, to blog, and so on, but as they started to realize being in "the PowerShell business" wasn't all that profitable, they let the site stagnate. Eventually, it became integrated with a larger, cross-topic community platform. This was my second lesson: _communities need to exist for their own reasons; commercial concerns will always, eventually, get distracted or change directions_. 
 
-I was also seeing a ton of great people doing amazing blogs on PowerShell, but they were all hard to find. Decent forums that didn't answer with a "you can just Google that" were few and far between. And so, in 2012, I decided to try again. 
\ No newline at end of file
+I was also seeing a ton of great people doing amazing blogs on PowerShell, but they were all hard to find. Decent forums that didn't answer with a "you can just Google that" were few and far between. And so, in 2012, I decided to try again. 
```

Comment:
Not sure why there is a change showing here, probably just BOF, although I made the edit exclusively in github.com so not sure why it would have changed that.